### ISSUE TITLE
Eyelids red

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9865]
**Preferred term label:**
Eyelids red

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Hordeolum

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
This is [Erythema of the eyelids HP:0040323](https://hpo.jax.org/app/browse/term/HP:0040323)

I am going to add Red eyelid as a synonym and also change the primary label to 
Eyelid erythema
